packages/terra-clinical-item-view/tests/wdio/item-view-spec.js

```diff
@@ -9,6 +9,16 @@ describe('Clinical Item View', () => {
     });
 
     Terra.should.matchScreenshot({ viewports, selector: '#test-displays' });
+
+    Terra.should.themeCombinationOfCustomProperties({
+      testName: 'themed',
+      selector: '#test-displays',
+      properties: {
+        '--terra-item-view-content-color': '#1441CA',
+        '--terra-item-view-content-primary-color': '#CA2BAC',
+        '--terra-item-view-content-secondary-color': '#D32416',
```

Comment:
Could these be green or blue, or something we can easily identify the color in the screenshots?